src/Components/Rooms/Rooms.tsx

```diff
@@ -1,7 +1,350 @@
-import React from 'react'
```

Comment:
This branch is also included and updated in a `rooms` related branch, you can consider delete it from here to avoid having a lot of conflicts.